pkgs/applications/audio/synaesthesia/default.nix

```diff
@@ -0,0 +1,30 @@
+{ stdenv
+, lib
+, fetchzip
+, SDL
+, pkg-config
+, libSM
+}:
+stdenv.mkDerivation rec {
+  pname = "synaesthesia";
+  version = "2.4";
+
+  src = fetchzip {
+    url = "https://logarithmic.net/pfh-files/synaesthesia/synaesthesia-${version}.tar.gz";
+    sha256 = "0nzsdxbah0shm2vlziaaw3ilzlizd3d35rridkpg40nfxmq84qnx";
+  };
+
+  buildInputs = [
+    SDL
+    pkg-config
```

Comment:
Shall we remove this (seems to build without it) or move this to nativeBuildInputs?

Reply:
Ah good catch! Fixed that :)